### PR TITLE
chore: fix svc show should list proper endpoint under "service discovery" section

### PR DESCRIPTION
**Pull request by bvtujo**
UI bug. svc show should display the env var as well as the "Service Discovery" section correctly. 

This PR adds a call to the env describer under the hood of the service describer clients to get the correct endpoint for the environment, service, and port. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

Screen Shot 2021-07-28 at 8 57 06 PM

### CODE REVIEW

File: internal/pkg/describe/backend_service.go [hunk 1]
```diff
@@ -48,8 +49,9 @@ func NewBackendServiceDescriber(opt NewBackendServiceConfig) (*BackendServiceDes
 		enableResources: opt.EnableResources,
 		store:           opt.DeployStore,
 		svcDescriber:    make(map[string]ecsSvcDescriber),
+		envDescriber:    make(map[string]envDescriber),
 	}
-	describer.initServiceDescriber = func(env string) error {
+	describer.initDescriber = func(env string) error {
```
Comment (efekarakus) [inline internal/pkg/describe/backend_service.go]: nit: 
```suggestion
	describer.initDescribers = func(env string) error {
```

File: internal/pkg/describe/backend_service.go [hunk 2]
```diff
@@ -112,12 +127,16 @@ func (d *BackendServiceDescriber) Describe() (HumanJSONStringer, error) {
 		}
 		port := blankContainerPort
 		if svcParams[cfnstack.LBWebServiceContainerPortParamKey] != cfnstack.NoExposedContainerPort {
+			endpoint, err := d.envDescriber[env].ServiceDiscoveryEndpoint()
+			// This error is descriptive and doesn't need to be wrapped.
```
Comment (efekarakus) [inline internal/pkg/describe/backend_service.go]: nit: can we remove the comment?

File: internal/pkg/describe/backend_service.go [hunk 3]
```diff
@@ -82,10 +93,14 @@ func (d *BackendServiceDescriber) URI(envName string) (string, error) {
 	if port == cfnstack.NoExposedContainerPort {
 		return BlankServiceDiscoveryURI, nil
 	}
+	envEndpoint, err := d.envDescriber[envName].ServiceDiscoveryEndpoint()
```
Comment (efekarakus) [inline internal/pkg/describe/backend_service.go]: nit: 
```suggestion
	endpoint, err := d.envDescriber[envName].ServiceDiscoveryEndpoint()
```

**Review by efekarakus — APPROVED** [review]
Looks good! Tiny nits that I can append on the PR while you're out. Thank you so much!

**Review by Lou1415926 — APPROVED** [review]

**Pull request merged**